source/libs/utils.ts

```diff
@@ -88,6 +88,10 @@ export const getOwnerAndRepo = (): {
 	return {ownerName, repoName};
 };
 
+export function getForkedRepo(): string | undefined {
+	return select<HTMLAnchorElement>('.fork-flag a')?.pathname.slice(1);
```

Comment:
Also you could use the same meta tags I used in the `isForkedRepo` since they're available earlier on the page:

```
<meta name="octolytics-dimension-repository_parent_nwo" content="sindresorhus/refined-github" />
```

```suggestion
 	return select<HTMLMetaElement>('[name="octolytics-dimension-repository_parent_nwo"]')?.content;
```

Reply:
Sure. 👍